app/models/hubstats/pull_request.rb

```diff
@@ -40,6 +40,18 @@ def self.create_or_update(github_pull)
       pull_data = github_pull.slice(*column_names.map(&:to_sym))
 
       pull = where(:id => pull_data[:id]).first_or_create(pull_data)
+
+      # Updates the merged_by part of the pull request and the user_id of the deploy
+      if github_pull[:merged_by] && github_pull[:merged_by][:id]
+        pull.merged_by = github_pull[:merged_by][:id]
+        pull_data[:merged_by] = pull.merged_by
```

Comment:
It only saves information that is in pull_data. If we don't add the new merged_by, then it won't save.